steering_controllers_library/src/steering_controllers_library.cpp

```diff
@@ -578,11 +585,25 @@ controller_interface::return_type SteeringControllersLibrary::update_and_write_c
 
     for (size_t i = 0; i < params_.traction_joints_names.size(); i++)
     {
-      command_interfaces_[i].set_value(timeout ? 0.0 : traction_commands[i]);
+      const auto value = timeout ? 0.0 : traction_commands[i];
+
+      if (!command_interfaces_[i].set_value(value))
+      {
+        RCLCPP_WARN(
+          logger, "Failed to set traction command %.3f for interface '%s' (index %zu).", value,
+          command_interfaces_[i].get_name().c_str(), i);
```

Comment:
like above, should we abort and return if one command fails?

Reply:
I would say return